### PR TITLE
If isFullScreen is true, resizes to the screen size rather than using iframe-resizer

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
https://git.dev.d2l/projects/CORE/repos/apploader/pull-requests/156/overview
https://git.dev.d2l/projects/CORE/repos/le/pull-requests/3422/overview

### CODE REVIEW

**Comment by dlockhart** [comment]
I'm kind of on the fence about this change now... like it's not truly going to be "full screen"... it's going to take up the "rest" of the screen height from where it was placed. So either we need to call that out in the option name (so that we could truly have a full screen option later), or try a different approach. 

I'm going to think about it more.

File: src/host.js [hunk 1]
```diff
@@ -33,7 +33,25 @@ export default class Host extends Port {
...
-			this.use(resizer);
+			if(options.isFullScreen) {
+				var me = this;
+
+				var handleResize = function() {
+					if(me.iframe) {
+						var rect = me.iframe.getBoundingClientRect();
+						me.iframe.style.height = (window.innerHeight - rect.top)+'px';
+					}
+				}
```
Comment (dlockhart) [inline src/host.js]: Missing a semi-colon here, which is why CI failed.

**Comment by dbatiste** [comment]
My 2 cents is that full screen means it takes up the entire screen.  I think using the entire space provided by the browser is different - perhaps full-page, or maybe something else.

**Comment by dougmoscrop** [comment]
Coming in to this with little context, it feels like it could be thought of as the equivalent of "position: absolute" I wonder if thinking of it in those terms is more intuitive than full scree/page.

**Comment by jstefaniuk-d2l** [comment]
This is now being changed out for the following: 
https://git.dev.d2l/projects/CORE/repos/lp/pull-requests/1786/overview
https://git.dev.d2l/projects/CORE/repos/le/pull-requests/3465/overview
https://git.dev.d2l/projects/CORE/repos/apploader/pull-requests/157/overview
https://github.com/Brightspace/ifrau/pull/54

**Pull request closed without merging**